src/lib/transition.ts

```diff
@@ -0,0 +1,24 @@
+export function showTitleWithMinimumScrolling() {
+  const headers = document.getElementsByTagName("h1")
+  if (headers.length) {
+    headers[0].scrollIntoView({
+      behavior: "smooth",
+      block: "nearest",
+      inline: "nearest",
+    })
+  }
+}
```

Comment:
Suggestion plus courte :

```suggestion
export function showTitleWithMinimumScrolling() {
  const header = document.querySelector("h1")
  header?.scrollIntoView({
    behavior: "smooth",
    block: "nearest",
    inline: "nearest",
  })
}
```